tools/cibadmin.c

```diff
@@ -775,9 +775,13 @@ main(int argc, char **argv)
     }
 
     rc = do_work(input, &output);
-    if (rc > 0) {
-        /* wait for the reply by creating a mainloop and running it until
-         * the callbacks are invoked...
+    if (!pcmk_is_set(options.cmd_options, cib_sync_call)
```

Comment:
Soon I'd like to make all CIB calls synchronous in `cibadmin`, unless we come up with a reason not to. Deprecate `--sync-call`. Not sure why we have these two separate code paths when the user running this from the command line has to wait for the call to receive a result either way.

Reply:
I believe it was originally for testing the code paths. Not needed anymore

Reply:
https://projects.clusterlabs.org/T817